Code/lib/corner/corner.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CORNER_H
+#define CORNER_H
+
+typedef enum{
+  LEFT,
+  RIGHT,
+  FORWARD,
+  BRAKE,
+}turn_direction;
+
+
+class CornerMerchantry
+{
+private:
+    bool wall_detected(float, float);
+    float *set_threshold_right;
+    float *set_threshold_front;
+public:
+    CornerMerchantry(float*, float*);
```

Comment:
You can name the parameters for readability

Reply:
it's a style preference, they're named in the .cpp file

Reply:
now done